src/org/sosy_lab/java_smt/test/ultimate/UltimateEliminatorWrapper.java

```diff
@@ -39,14 +45,23 @@ public UltimateEliminatorWrapper(LogManager pLog) {
     log = pLog;
   }
 
+  /*
+   * Simplifies and try to remove the quantifiers from the given term using the UltimateEliminator.
+   */
   public Term simplify(Term pTerm) {
     return ultimateEliminator.simplify(pTerm);
```

Comment:
UE has lots of options that allow different types of simplifications.
Do we have a way of changing those?